csharp/src/Ice/Proxy.cs

```diff
@@ -559,5 +538,23 @@ public static class Proxy
                 }
             }
         }
+
+        private class ProgressWrapper : IProgress<bool>
+        {
+            private readonly IProgress<bool>? _progress;
+
+            public bool IsSent { get; private set; }
```

Comment:
should this be `internal` instead of `public`?

Reply:
It should be internal